src/current/_includes/v23.2/prod-deployment/healthy-lsm.md

```diff
@@ -1 +1 @@
-**Expected values for a healthy cluster**: The number of L0 files should **not** be in the high thousands. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.
\ No newline at end of file
+**Expected values for a healthy cluster**: An IO Overload value greater than 1.0 generally indicates an overload in the Pebble LSM tree. High values indicate heavy write load that is causing accumulation of files in level 0. These files are not being compacted quickly enough to lower levels, resulting in a misshapen LSM.
```

Comment:
"misshapen LSM" could be a link to https://www.cockroachlabs.com/docs/v23.2/architecture/storage-layer#inverted-lsms